contracts/common/lib/Merkle.sol

```diff
@@ -10,10 +10,11 @@ library Merkle {
   ) public pure returns (bool) {
     bytes32 proofElement;
     bytes32 computedHash = leaf;
-    uint256 len = (proof.length / 32) * 32;
+    require(proof.length % 32 == 0, "Invalid proof length");
+    // uint256 len = (proof.length / 32) * 32;
```

Comment:
Thanks for keeping the comment explained stuff, I think now we can remove :d

Reply:
Done.